crates/pipeline-manager/src/lib.rs

```diff
@@ -70,3 +71,10 @@ pub fn init_fd_limit() {
         }
     }
 }
+
+/// Check if a pipeline compiled for the specificed `runtime_version` is compatible with the current `platform_version`.
+// TODO: this is a placeholder implementation for now.
+// TODO: do we need `platform_version` here at all? This can be implicitly the current running platform version.
```

Comment:
yes shouldn't be needed see /v0/config implements on how to get the current platform version